tests/service/privacy_request/request_runner_service_test.py

```diff
@@ -172,6 +173,57 @@ def test_create_and_process_access_request(
     assert ExecutionLog.get(db, id=log_id).privacy_request_id == pr_id
 
 
+@pytest.mark.integration
+@mock.patch("fidesops.models.privacy_request.PrivacyRequest.trigger_policy_webhook")
+def test_create_and_process_access_request_mssql(
+        trigger_webhook_mock,
+        mssql_example_test_dataset_config,
+        db,
+        cache,
+        policy,
+        policy_pre_execution_webhooks,
+        policy_post_execution_webhooks,
+):
+
+    customer_email = "customer-1@example.com"
+    data = {
+        "requested_at": "2021-08-30T16:09:37.359Z",
+        "policy_key": policy.key,
+        "identity": {"email": customer_email},
+    }
+
+    pr = get_privacy_request_results(db, policy, cache, data)
+
+    results = pr.get_results()
+    assert len(results.keys()) == 11
+
+    for key in results.keys():
+        assert results[key] is not None
+        assert results[key] != {}
+
+    result_key_prefix = f"EN_{pr.id}__access_request__mssql_example_test_dataset:"
+    customer_key = result_key_prefix + "customer"
+    assert results[customer_key][0]["email"] == customer_email
+
+    visit_key = result_key_prefix + "visit"
+    assert results[visit_key][0]["email"] == customer_email
+    log_id = pr.execution_logs[0].id
+    pr_id = pr.id
+    # Both pre-execution webhooks and both post-execution webhooks were called
+    assert trigger_webhook_mock.call_count == 4
+
+    for webhook in policy_pre_execution_webhooks:
+        webhook.delete(db=db)
+
+    for webhook in policy_post_execution_webhooks:
+        webhook.delete(db=db)
+
+    policy.delete(db=db)
```

Comment:
If we delete fixtures outside of their teardown methods this can lead to errors depending on the order the tests run in, we should remove this. The same for those `webhook.delete()` lines above too

Reply:
Hmm we were doing this before, in `test_create_and_process_access_request`, unsure if @pattisdr has any more context into why we're manually deleting webhooks vs using fixtures? My fixture knowledge isn't strong here.

Reply:
It's more relevant to `session` level fixtures, where if we delete them before they're due to be re-used they won't get created again between tests.

<img width="568" alt="Screenshot 2022-01-20 at 13 20 40" src="https://user-images.githubusercontent.com/1667340/150398236-04ec6b78-daa4-41ee-9de9-f09b41e1915c.png">

In this instance we'd be able to avoid nested the delete inside `try / except` blocks, which is better because test data creation and deletion should happen consistently in the same place where possible

Reply:
@eastandwestwind, it looks like in `test_create_and_process_access_request`, we're not deleting the privacy request for cleanup purposes, we're trying to verify a consequence of deleting the privacy request - in other words, the deletion was part of the test.  We're checking that even in the event a privacy request is removed, its execution logs still remain for record-keeping.  Generally, we should be relying on fixture teardown.  

Perhaps this means for `test_create_and_process_access_request` too, we should create a separate test that just creates the resources in the test and then deletes them, so we're not tearing down a fixture.

Reply:
Got it, thanks, @seanpreston and @pattisdr for the clarification! 

For the new `test_create_and_process_access_request_mssql` test, I don't necessarily have to assert for logs after deleting privacy request, so I can just remove that bit (L:215-224).